src/test/rustdoc/issue-83832.rs

```diff
@@ -0,0 +1,21 @@
+#![crate_name = "foo"]
```

Comment:
"issue-xxxxx" doesn't help people know what this is testing - could you rename it to "reexport-stability-tags" or something?

Reply:
Done.